### ISSUE TITLE
Error: credentialsService.createCredential - Malformed UTF-8 data

### ISSUE THREAD

**Issue by saikumar-digi** [FlowiseAI/Flowise#2467]
Sometimes when we click on edit createCredential that time I get 500 internal server error with Error: credentialsService.createCredential - Malformed UTF-8 data i connected Mysql Data base locally its working I'm using GCP DB there I'm getting this error

**Comment by arthaadiadayamadani** [comment]
Same i getting the error by using postgresql as the DB. 
Error : 
Status: 500
Error: credentialsService.createCredential - Malformed UTF-8 data

**Comment by H1bertto** [comment]
Same here, I am also using GCP with PostgreSQL, in the credentials I get this error:
`Status: 500 Error: credentialsService.createCredential - Malformed UTF-8 data`
And if this error occurs in the credential, in Chatflow I get this error:
`Error: Error: Credentials could not be decrypted.`

**Comment by arakakihenrique** [comment]
same here, but for openai.

**Comment by sjoel99** [comment]
I also have the same problem

**Comment by yuralves** [comment]
Same issue here.
<img width="1135" alt="image" src="https://github.com/user-attachments/assets/52893a6d-1626-418f-a0be-1cfa024867c5">
Can't find a root cause. I created two identical credentials on the same day and i'm getting error with one (after a few hours of creation) and success with the other

**Comment by frankyw** [comment]
Same problem here, creating an OpenAI credential with Supabase backend.

**Comment by sandroelmo** [comment]
same problem, PostgreSQL on cloud run. Any solution?

**Comment by josh-at-straker** [comment]
bump - this is major - also breaking all our production applications.

**Comment by hhaweb** [comment]
yes same issue after I restart the flowise(I deploy in Kubernetes), can't click on edit button but I can add new one and when I click edit on new one , there is no issue 

Error: credentialsService.createCredential - Malformed UTF-8 data

**Comment by SleepingSmel** [comment]
> I found that I had to set `SECRETKEY_PATH` as well as `FLOWISE_SECRETKEY_OVERWRITE` to fix this, when deployed as a container. I set it as `SECRETKEY_PATH=/root/.flowise` as per the README here https://github.com/FlowiseAI/Flowise/tree/main/docker

But this is for local databases, not online mysql or postgres databases, right? Do those parameters have to be set as variables, or added in the docker file? Thanks!

**Comment by localLoopers** [comment]
Error: Malformed UTF-8 data  Q: what the fix  for this

**Comment by juancarlossch** [comment]
> Error: Datos UTF-8 mal formados P: cuál es la solución para esto

Tengo el mismo error

**Comment by bitsnaps** [comment]
Right now, the only option is to delete/re-create credentials every time. I think this can be solved in the implementation provide an option to store var env in the DB instead, well this should be encrypted though.
